### PR TITLE
Issue #11214: minor: Specified violation messages in CustomImportOrder

### PR DESCRIPTION
**Pull request by aaryan-gautam**
Fixes https://github.com/checkstyle/checkstyle/issues/11214

### CODE REVIEW

**Comment by romani** [comment]
Please do fixes of violations of CI

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderDefault.java [hunk 1]
```diff
@@ -13,21 +13,23 @@
...
-import java.awt.Dialog; // violation
-import java.awt.color.ColorSpace; // violation
-import java.awt.event.ActionEvent; // violation
-import javax.swing.JComponent; // violation
-import javax.swing.JTable; // violation
-import java.io.File; // violation
-import java.io.IOException; // violation
-import java.io.InputStream; // violation
-import java.io.Reader; // violation
+import java.awt.Button; // violation should be at the end
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderDefault.java:22]: Please use real violation message.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderDefault.java (hunk 1 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderDefault.java:22]: Please review changes in similar Check https://github.com/checkstyle/checkstyle/pull/12772/files

**Review by romani — CHANGES_REQUESTED** [review]
Good, but please remove this Check from suppression list in java code, see issue description to a line where to do this.

**Comment by aaryan-gautam** [comment]
> Good, but please remove this Check from suppression list in java code, see issue description to a line where to do this.

I have already removed the check from suppression list in the previous commit 04ed952584bbb6bc08ca11c73c6f848f32fd1075
https://github.com/checkstyle/checkstyle/blob/4838dbc82ced038cfb72fdcada612dcfdec8a93c/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L105

**Comment by romani** [comment]
Please keep single commit in PR all the time.
Please strictly follow commit message template, please fix CI failures

**Comment by rnveach** [comment]
CI needs to be green. Needs to be rebased.

**Comment by romani** [comment]
Considering amount of conflicts, better to do it from scratch

**Pull request closed without merging**